web3-functions/gelato-cctp/api.ts

```diff
@@ -0,0 +1,52 @@
+import { GELATO_API, CIRCLE_API } from "../../src/cctp-sdk/constants";
+import { CallWithSyncFeeRequest } from "@gelatonetwork/relay-sdk";
+import {
+  IAttestation,
+  AttesationState,
+  IRelayRequestResponse,
+  IRelayTaskStatusResponse,
+} from "./types";
+import ky from "ky";
+
+const getErrorMsg = (error: unknown) =>
+  error instanceof Error ? error.message : String(error);
+
+export const getRelayTaskStatus = async (taskId: string) => {
```

Comment:
Define return type and make sure TSLINT is properly configured as the Linting step on Husky should have FAILED when you commited.